hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/DeleteBlocksCommandHandler.java

```diff
@@ -639,8 +639,13 @@ private void updateMetaData(KeyValueContainerData containerData,
 
       // update pending deletion blocks count and delete transaction ID in
       // in-memory container status
+      long pendingBytes = delTX.getTotalBlockSize();
       containerData.updateDeleteTransactionId(delTX.getTxID());
-      containerData.incrPendingDeletionBlocks(newDeletionBlocks);
+      containerData.incrPendingDeletionBlocks(newDeletionBlocks, pendingBytes);
+      metadataTable
```

Comment:
parallel thread can update same key, deleteCommandHandler and Blockdeletingservice, need use some lock with container as being used for update containerdata.